pack.go

```diff
@@ -126,6 +128,11 @@ var mediaTypeRegexp = regexp.MustCompile(`^[A-Za-z0-9][A-Za-z0-9!#$&-^_.+]{0,126
 //
 // artifactType and opts.ConfigDescriptor.MediaType MUST comply with RFC 6838.
 //
+// Each time when PackManifest is called, it generates a new time stamp in the
+// manifest annotations with the key ocispec.AnnotationCreated. To make
```

Comment:
nit: Do we need to explicitly call out that the timestamp is generated only when `ocispec.AnnotationCreated` annotation is not specified?

Reply:
Modified the comment